.github/workflows/ccpp.yml

```diff
@@ -39,7 +39,7 @@ jobs:
                   g++-multilib \
                   dejagnu;
           # install Rust directly using rustup
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=1.72.0;
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=nightly;
```

Comment:
I'm curious. Why do we need to use nightly instead of latest stable?

Reply:
because I needed to use rust extern types (opaque types) [here](https://github.com/Rust-GCC/gccrs/pull/3076/commits/e8641d4d492ca20cca4fd51332369fa2dfacc60d#diff-0258361e923e1e940fda21f39e64ea438a3a54d480985f587c5bef2407932589R35-R39)
its an ustable feature, so not supported in stable rust currently